### PR TITLE
libraries: improve display of notification settings

### PR DESCRIPTION
**Pull request by Garfield-fr**
Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

## Why are you opening this PR?

- improve display of notifications

notifications

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

**Comment by iGormilhit** [comment]
Commit title → ` libraries: improve display of notification settings`

**Review by iGormilhit — APPROVED** [review]
Commit message approved.

File: projects/admin/src/app/record/detail-view/library-detail-view/library-detail-view.component.html [hunk 1]
```diff
@@ -75,19 +75,18 @@ <h1>{{ record.metadata.name }}</h1>
...
-            <dd class="col-sm-7 col-md-8 mb-0">{{ setting.email }}</dd>
-            <!-- DELAY AVAILABILITY -->
-            <dt class="col-sm-3 offset-sm-2 offset-md-0" *ngIf="setting.delay">{{ 'Delay' | translate }}</dt>
-            <dd class="col-sm-7 col-md-8 mb-0">{{ setting.delay }}</dd>
-          </dl>
-        </section>
-      </article>
+      <ul class="list-unstyled" *ngIf="record.metadata.notification_settings as notificationSettings">
+        <li class="row" *ngFor="let setting of notificationSettings">
+          <dt class="col-sm-3 offset-sm-2 offset-md-0 font-weight-bold">{{ setting.type | ucfirst | translate }}:</dt>
```
Comment (zannkukai) [inline projects/admin/src/app/record/detail-view/library-detail-view/library-detail-view.component.html:80]: `class="offset-2 col-3"` for "DT" tag and `class="col-7"` for "DD" tag

**Review by zannkukai — APPROVED** [review]

File: projects/admin/src/app/record/detail-view/library-detail-view/library-detail-view.component.html (hunk 1 above)
Comment (Garfield-fr) [reply projects/admin/src/app/record/detail-view/library-detail-view/library-detail-view.component.html]: done

**Review by benerken — APPROVED** [review]

**Review by lauren-d — APPROVED** [review]

**Pull request merged**